src/api/Tag.php

```diff
@@ -15,7 +15,9 @@ class Tag
     const RESOURCE_NAME = 'resource.name';
     const DB_STATEMENT = 'sql.query';
     const ERROR = 'error';
-    const ERROR_MSG = 'error.msg'; // string representing the error message
+    const ERROR_MESSAGE = 'error.message'; // string representing the error message
+    /** @deprecated Use ERROR_MESSAGE instead. */
+    const ERROR_MSG = 'error.msg';
```

Comment:
We can migrate the value here, I think?

```suggestion
    const ERROR_MSG = 'error.message';
```